### PR TITLE
[CALCITE-3422] Reopen tests in PigRelBuilderStyleTest.

### PR DESCRIPTION
**Pull request by xy2953396112**
reOpen CALCITE-1751 tests in PigRelBuilderStyleTest.

### CODE REVIEW

**Comment by amaliujia** [comment]
Nice finding! 

LGTM

**Comment by yanlin-Lynn** [comment]
As described in [CALCITE-1751](https://issues.apache.org/jira/browse/CALCITE-1751)
` It seems like it fails every 2 or 3 builds which causes the jobs to constantly flip from failing to stable`. 
Better to make sure the test works stably before reopen these tests.

File: pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java [hunk 1]
```diff
@@ -84,7 +83,6 @@ public void testScanAndFilter() throws Exception {
   }
 
   @Test
-  @Ignore("CALCITE-1751")
   public void testImplWithMultipleFilters() {
```
Comment (danny0405) [inline pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: Did the fail reason in CALCITE-1751 has been resolved ? I didn't see any one dug into that , can you give more details ?

Comment (xy2953396112) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: As described in CALCITE-1751, run those tests in my machine many times, run 'testUserDefinedFunction' many times and it works correctly.

Comment (danny0405) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: Not in your machine, you should validate this on the travis machine.

Comment (xy2953396112) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: not sure why those tests failed before , run  tests on the travis machine and and it works correctly and stablely.

Comment (danny0405) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: I have no permission for the travis machine, this bug is not stable, it happens after multiple rounds of tests. I have no chance to validate this and there is no evidence that this bug is fixed.

Comment (xy2953396112) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: Very embarrassed，is there someone can validate this?

Comment (zabetak) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: There two ways to check if the tests are still failing:
* Commit on master and wait the results of Jenkins; if not fixed rollback the commit.
* Create a separate branch, commit there, create a new Jenkins job to follow that branch and check after various runs if everything is good.

The second requires more effort than the time spend for this PR and I don't have much while the first might reduce unnecessary noise to the master.

Comment (zabetak) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: @danny0405 ask in the dev list to get permissions to control the Jenkins jobs. See https://lists.apache.org/thread.html/6a9d873206ca8965d24f91236151f195f4c861aa21aed7dc374b2259@%3Cdev.calcite.apache.org%3E

Comment (danny0405) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: @zabetak Thanks for giving the suggestion, i have sent the mail :)

Comment (julianhyde) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: Hey @danny0405, totally unrelated to this bug, but on the subject of email, did you know that @F21 has been trying to contact you via email on behalf of the PMC? Can you please email him or the private list?

Comment (danny0405) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: @julianhyde , oh, sorry for that, i just saw the email, i'm so sorry for the delay @F21 , i'm glad to join you and it's my honor to work with you  !

Comment (xy2953396112) [reply pig/src/test/java/org/apache/calcite/test/PigRelBuilderStyleTest.java]: > @julianhyde , oh, sorry for that, i just saw the email, i'm so sorry for the delay @F21 , i'm glad to join you and it's my honor to work with you !

Congratulations！

**Comment by xy2953396112** [comment]
@danny0405  Thanks, is there some problems in  travis machine? and this bug is fixed?

**Comment by danny0405** [comment]
@xy2953396112 I have no time to check that yet, sorry :( Maybe i would try it this weekend.

**Comment by xy2953396112** [comment]
Is there some problems in travis machine ?  Thanks.

**Pull request closed without merging**